packages/xod-arduino/src/templates.js

```diff
@@ -114,15 +115,26 @@ Handlebars.registerHelper('global', function global(options) {
 
 Handlebars.registerHelper('cppType', type => cppType(type));
 
-// Converts JS-typed data value to a string that is valid and expected
-// C++ literal representing that value
+// Converts bound value literals or JS-typed data value to a
+// string that is valid and expected C++ literal representing that value
 Handlebars.registerHelper('cppValue', (type, value) =>
-  R.propOr(R.always('unknown_type<void>'), type, {
-    [PIN_TYPE.PULSE]: R.always('false'),
-    [PIN_TYPE.BOOLEAN]: R.toString,
-    [PIN_TYPE.NUMBER]: R.toString,
-    [PIN_TYPE.STRING]: cppStringLiteral,
-  })(value)
+  R.compose(
+    R.propOr(R.always('unknown_type<void>'), type, {
+      [PIN_TYPE.PULSE]: R.always('false'),
+      [PIN_TYPE.BOOLEAN]: R.ifElse(
+        R.equals('True'),
+        R.always('true'),
+        R.always('false')
+      ),
+      [PIN_TYPE.NUMBER]: R.identity,
+      [PIN_TYPE.STRING]: R.compose(
+        cppStringLiteral,
+        R.replace(/"/g, '\\"'),
+        unquote
+      ),
```

Comment:
The function exploded a bit. `R.replace(/"/g, '\\"')` should semantically belong to `cppStringLiteral`.

The repetitive code for `BOOLEAN`, perhaps, better expressed with a simple ternary.